src/firefly/js/ui/VoSearchPanel.jsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+import React, {Component, PropTypes} from 'react';
+import {TargetPanel} from '../ui/TargetPanel.jsx';
+import {SizeInputFields, sizeFromDeg} from './SizeInputField.jsx';
+import FieldGroupUtils from '../fieldGroup/FieldGroupUtils.js';
+import {ListBoxInputField} from './ListBoxInputField.jsx';
+import './VoSearchPanel.css';
+
+export class VoSearchPanel extends Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    componentWillUnmount() {
+        if (this.removeListener) this.removeListener();
+        this.iAmMounted = false;
+    }
+
+    componentDidMount() {
+        this.iAmMounted = true;
+        this.removeListener = FieldGroupUtils.bindToStore('CATALOG_PANEL', (fields) => {
+            if (this.iAmMounted) this.setState(fields);
+        });
+    }
+
+    render() {
+        const fields = this.state;
+        return (
+            <div className={'vopanel'}>
+                <div className={'section'}>
+                    {targetPanelArea()}
+                </div>
+                <div className={'size'}>
+                    { sizeArea()}
+                </div>
+                <div className={'voarea'}>
+                    { voSearchArea() }
+                </div>
+            </div>
+        );
+
+    }
+
+}
+
+function targetPanelArea() {
+    return (
+        <div className={'intarget'}>
+            <TargetPanel groupKey={'CATALOG_PANEL'}/>
+            <ListBoxInputField
```

Comment:
This will be part of Target panel. I just have not done it yet.  You should take out this list box.